src/morello-install/README.md

```diff
@@ -640,28 +651,24 @@ If there are further configuration settings to change, select an appropriate
 option from the menu and press Space.
```

Comment:
This needs updating now the Exit entry has moved to its own separate Finish button